ocs/defaults.py

```diff
@@ -4,6 +4,7 @@
 from getpass import getuser
 
 INSTALLER_VERSION = '4.1.0-rc.3'
+CLIENT_VERSION = INSTALLER_VERSION
```

Comment:
Define this in default_config.yaml in RUN section, I think it's part of RUN and won't be used anywhere else I think. Anyway it can be taken from RUN section from ocsci/config.py module from whatever place

Reply:
Can you link what file are you talking about here?

Reply:
@vasukulkarni was talking about this file: https://github.com/red-hat-storage/ocs-ci/blob/master/conf/ocsci/default_config.yaml . But I am trying to solve some problem which @dahorak  mentioned in his another comment which is not easy fix cause of import modules before we are actually loading config in some cases.